ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Build and deploy the controller image..."
+./ci/github/build_controller
+
+echo "Build and deploy the DriverContainer..." 
```

Comment:
Isn't it just building? Anyway it should be part of the infra, otherwise we are going to re-build an already built image m/s.